event-consumer/actions/generic/index.js

```diff
@@ -0,0 +1,148 @@
+/*
+* <license header>
+*/
+
+/**
+ * This is a sample action showcasing how to access an external API
+ *
+ * Note:
+ * You might want to disable authentication and authorization checks against Adobe Identity Management System for a generic action. In that case:
+ *   - Remove the require-adobe-auth annotation for this action in the manifest.yml of your application
+ *   - Remove the Authorization header from the array passed in checkMissingRequestInputs
+ *   - The two steps above imply that every client knowing the URL to this deployed action will be able to invoke it without any authentication and authorization checks against Adobe Identity Management System
+ *   - Make sure to validate these changes against your security requirements before deploying the action
+ */
+
+const fetch = require('node-fetch')
+const { Core, Events } = require('@adobe/aio-sdk')
+const { context, getToken } = require('@adobe/aio-lib-ims')
+const stateLib = require('@adobe/aio-lib-state')
```

Comment:
a simpler way to import State SDK:
```
const { Core, Events, State } = require('@adobe/aio-sdk')
```
Then you don't need to specify aio-lib-state separately in package.json

Reply:
done